macro/src/main/scala-2.x/monocle/macros/syntax/AppliedFocusSyntax.scala

```diff
@@ -27,10 +27,20 @@ class GenAppliedLensOpsImpl(val c: blackbox.Context) {
         c.abort(c.enclosingPosition, s"Invalid prefix tree ${show(t)}")
     }
 
+    lambda match {
+      // _.field
+      case Expr(Function(List(ValDef(_, _, _, EmptyTree)), Select(Ident(_), _))) => ()
```

Comment:
This is really ugly to duplicate code from `GenLens` macro but I couldn't find out how to pass a Boolean flag ...

I don't think it matters too much since it is for Scala 2 and we don't plan too touched macro further in this version of Scala.